docs/release/releases/release-1.7.md

```diff
@@ -0,0 +1,36 @@
+# Cluster API v1.7
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.7` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 4th December 2023    | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 8th December 2023    | week 1   |
+| Team finalized                                       | Release Lead | Friday 8th December 2023    | week 1   |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 9th January 2024    | week 6   |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 6th February 2024   | week 10  |
+| v1.7.0-beta.0 released                               | Release Lead | Tuesday 5th March 2024      | week 14  |
+| Communicate beta to providers                        | Comms Manager| Tuesday 5th March 2024      | week 14  |
+| *v1.5.x & v1.6.x released*                           | Release Lead | Tuesday 5th March 2024      | week 14  |
+| v1.7.0-beta.x released                               | Release Lead | Tuesday 12th March 2024     | week 15  |
+| KubeCon idle week                                    |      -       | 19th - 22nd March 2024      | week 16  |
+| release-1.7 branch created (**Begin [Code Freeze]**) | Release Lead | Tuesday 26th March 2024     | week 17  |
```

Comment:
I'm a bit concerned with the code freeze date falling right after kubecon, where maintainers/reviewers will come back after a week away to an inbox full of notifications and there will a lot of last minute PRs that want to get in before code freeze... Not sure if there is another way

Reply:
We're doing something similar for this current release with Kubecon NA so we can see how that works out I guess?

Reply:
@CecileRobertMichon that is right, it is the same for the current release cycle and this was kind of discussed when adding v1.6 timeline: https://github.com/kubernetes-sigs/cluster-api/pull/8901#discussion_r1238258493

Reply:
I think my point is that if this current release cycle doesn't work out well with the code freeze right after kubecon we should consider amending the timeline for 1.7, but we won't know until after we do the code freeze for 1.6.

Reply:
As discussed in today's (1/11/23) office hours and pointed out by @CecileRobertMichon, we will see how it works out with the current release cycle and decide based on that for the 1.7 release.

Reply:
Just wanted to add that there's also the easter weekend ~ from 29 March - 1 April

Reply:
But looks like we already have a 19 weeks cycle, we probably shouldn't extend this further otherwise the other two release cycles will become even shorter in 2024